react/Badge/Badge.js

```diff
@@ -0,0 +1,53 @@
+// @flow
+
+import React, { PureComponent } from 'react';
+import classnames from 'classnames';
+
+import styles from './Badge.less';
+
+type Props = {
```

Comment:
We've used a `tone` prop in `Section` for critical, info, positive etc. https://github.com/seek-oss/seek-style-guide/blob/master/react/Section/Section.js#L15. Not saying this is wrong but we should try and have the same API for both components.